src/main/java/org/elephant/sam/SAMExtension.java

```diff
@@ -108,9 +110,10 @@ private SAMCommands(QuPathGUI qupath) {
 				qupath.getImageData().getHierarchy().removeListener(this);
 			});
 			actionSetServerURL = new Action(event -> {
-				String newURL = Dialogs.showInputDialog("Server URL", "Set API server URL", serverURL);
+				String newURL = Dialogs.showInputDialog(
```

Comment:
There is also an option to put this information in the preferences dialog.
Which one do you recommend?
I had a PR by @rharkes.
https://github.com/rharkes/qupath-extension-sam/blob/81f9246cc44af78f705747e640862546eefde442/src/main/java/org/elephant/sam/SAMExtension.java#L66-L68

Reply:
The later commits basically removed this anyway, and put it in the main dialog with everything else.

I think adding to the preferences is good if most people never need to know the option exists or change its values. But for something as important as this, I think it shouldn't be *only* in the preferences - it should be somewhere more obvious in the UI. Then it could be added to the preferences as well, but doesn't have to be.

Reply:
That makes sense. Thank you for your comments and suggestions!